src/jats/jats-exporter.ts

```diff
@@ -1067,9 +1059,13 @@ export class JATSExporter {
       hard_break: () => '',
       highlight_marker: () => '',
       inline_footnote: (node) => {
+        const rids = node.attrs.rids.filter(getModel)
```

Comment:
Sometimes we receive an empty table-wrap-foot, however, table content references a footnote that doesn't exist.